pom.xml

```
@@ -43,8 +43,8 @@
         <jboss.snapshots.repo.id>jboss-snapshots-repository</jboss.snapshots.repo.id>
         <jboss.snapshots.repo.url>https://s01.oss.sonatype.org/content/repositories/snapshots/</jboss.snapshots.repo.url>
 
-        <quarkus.version>3.2.4.Final</quarkus.version>
-        <quarkus.build.version>3.2.3.Final</quarkus.build.version>
+        <quarkus.version>3.2.5.Final</quarkus.version>
+        <quarkus.build.version>3.2.5.Final</quarkus.build.version>
```

Do we generally want to set these individually?  If not, we could just use:

```suggestion
        <quarkus.build.version>${quarkus.version}</quarkus.build.version>
```

---

IIRC, there was some reason for this for productization. @ASzc Can you please confirm?

---

It's because some of the quarkus build-time components broke when using aligned versions. Splitting the property into runtime/buildtime allows not aligning the build-time parts. I tried the method shown here, but PME sees it as one property. Instead I made sure `quarkus/set-quarkus-version.sh` updated both properties, but I guess people haven't been using that?

I'm not sure if the root problem still exists, but it's good practice not to share properties between build time and runtime dependencies. Quarkus is kinda unusual in that it has a lot of runtime and buildtime stuff.